docs/source/docker/getting_started/first_steps.rst

```diff
@@ -65,8 +65,8 @@ There are **three** types of volume mappings:
    See the tutorial :ref:`input-structure-label`  for more information.
    The container has only **read access** to this directory (note the *:ro* at
    the end of the volume mapping).
-   Instead of using a local input directory you can also use a S3 bucket as a remote datasource.
-   For reference, head to :ref:`ref-docker-with-datasource`.
+   Instead of using a local input directory you can also use a cloud bucket as a 
```

Comment:
To spice up searching, maybe;
a cloud bucket as a -> a cloud storage bucket on S3, GCS or Azure as a remote datasource